ext/mbstring/libmbfl/filters/mbfilter_utf16.c

```diff
@@ -27,15 +27,159 @@
  *
  */
 
+#include <stdint.h>
+#include <stdbool.h>
+#include <string.h>
+#include "zend_portability.h"
+#include "zend_long.h"
+#include "zend_bitset.h"
```

Comment:
Here I really just wanted `zend_bitset.h`, but had to include the preceding headers for `zend_bitset.h` to work. Is there some other header which I should include instead, which includes all of these?

Reply:
Right, the issue is ``zend_bitset.h`` doesn't include the standard headers it uses, might make sense to fix that there instead of here?

Reply:
Good idea.